docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -141,7 +136,8 @@ with the current date and the property updated, we can do:
 
 [source,cypher]
 ----
-CALL apoc.trigger.add('setLastUpdate',
+USE system 
+CALL apoc.trigger.install('neo4j', 'setLastUpdate',
```

Comment:
Does it work with Fabric like this? Or do you mean to say `:use system`? 

`USE system` is not equivalent to `:use system` because the first relies on Fabric routing and the latter relies on drivers routing. 

I couldn't see any tests where we are using Fabric to route to the system database, maybe that's possible but if so then I'd like to be sure. If not then maybe it's better to use `:use system` here instead?

Reply:
Yes, both `:use system` and `USE system CALL apoc.trigger.install`, with a `neo4j://` protocol, works.

I didn't add tests for fabric https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3183#discussion_r1018198006 

But maybe we can make the documentation more generic: that is, something like putting a `[NOTE]` at the beginning saying that these procedures have to be used via `":use system"`, by opening a session in "system" db with neo4j driver, or via `"USE system ..."` .

And as an example here just put `CALL apoc.trigger.install('neo4j', 'setLastUpdate', ...`.
This way it is up to the user what he wants to use. 
Wdyt?

Reply:
Ah yes good point I remember that conversation now! 

I think your suggestion is good 👍